src/core_modules/capture-core/metaData/RenderFoundation/CustomForm.js

```diff
@@ -26,56 +26,22 @@ export class CustomForm {
     set id(id: string) {
         this._id = id;
     }
-    get id() {
+    get id(): string {
         return this._id;
     }
     /**
      * parses html to react elements
      *
      * @memberof CustomForm
      */
-    set data(html: string) {
-        const data = parseHtml(html, {
-            onTransform: this.transformNode,
+    setData(html: string, transformFunction: (node: Object, index: number) => ?Element<'FormField'>) {
```

Comment:
```suggestion
   setData(html: string, transformFunction: TransformFunction) {
```

and the type definition: `type TransformFunction =
    (node: Object, index: number, nodeToElementFn: (node: Object, index: number) => Element<'FormField'>) =>
        void | Element<'FormField'>;` (it's ok to put this in the CustomForm.js at the top).

Should fix the Flow error you have in the EnrollmentFactory (line 176)

Reply:
Thanks, @JoakimSM! 👍 Implemented.